pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -564,3 +609,10 @@ func memberInList(seek string, a ...string) bool {
 	}
 	return false
 }
+
+func isNominalConcurrencyShareFieldDefault(obj *flowcontrol.PriorityLevelConfiguration) bool {
```

Comment:
`isNominalConcurrencyShareFieldDefault` is not exactly the right name for this function. This function does not test whether that field has its default value --- which may be regarded as `nil` or a pointer to `int32(30)`. What this function tests is whether is object is of the Limited sort an the NominalConcurrencyShares is a pointer to zero.